pkg/cloud/utils.go

```diff
@@ -128,9 +139,25 @@ func (r *ResourceID) String() string {
 //	[https://www.googleapis.com/compute/<ver>]/projects/<proj>/global/<res>/<name>
```

Comment:
We should update the docstring

Reply:
Updated all of these to have `<apigroup>` instead of `compute`